admin/app/components/solidus_admin/adjustment_reasons/index/component.rb

```diff
@@ -17,7 +17,10 @@ def page_actions
     render component("ui/button").new(
       tag: :a,
       text: t('.add'),
-      href: solidus_admin.new_adjustment_reason_path, data: { turbo_frame: :new_adjustment_reason_modal },
+      href: solidus_admin.new_adjustment_reason_path, data: {
+        turbo_frame: :new_adjustment_reason_modal,
+        turbo_prefetch: false
```

Comment:
btw we probably can turn `turbo-prefetch` off [globally with meta tag](https://turbo.hotwired.dev/handbook/drive#prefetching-links-on-hover)
```
<meta name="turbo-prefetch" content="false">
```
I can see prefetching is done on almost all the links, but the problem with it is that prefetch is done on each and every hover on the same link, i.e you hover five times over one link and it makes five requests to the same endpoint. It's not really useful without caching and if i'm not mistaken there's no caching set up (but please correct me if i'm wrong!)

Reply:
Good catch. We should disable it for the whole admin. There is no caching in place

Reply:
Done